### PR TITLE
openblas on mac + llvm + go over readme

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -119,14 +120,38 @@ before_install:
...
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - mkdir -p build && cd build
   - |
     if [[ $BUILD_PACKAGE == "ON" && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
-      export CMAKE_OPTIONS="-DCMAKE_CXX_STANDARD=11 -DBLA_VENDOR=$BACKEND"
+      export CMAKE_OPTIONS=" -DBLA_VENDOR=$BACKEND"
     else
-      export CMAKE_OPTIONS="-DCMAKE_CXX_STANDARD=11 -DBLA_VENDOR=$BACKEND -DENABLE_COVERAGE=ON"
+      export CMAKE_OPTIONS=" -DBLA_VENDOR=$BACKEND -DENABLE_COVERAGE=ON"
```
Comment (agramfort) [inline .travis.yml]: @massich I remove this as set it in the cmake. no need to add this to cmkae line all the time.

Comment (massich) [reply .travis.yml]: I disagree, I would rather take it out from cmake. But I can do it in a PR when adding specific compilation flags.

Comment (agramfort) [reply .travis.yml]: pb is that cmake then does not work out of the box and you need cleverness to be able to build...

Comment (agramfort) [reply .travis.yml]: ideally doing:

mkdir build
cd build
cmake .
make

should work provided you have installed the dependencies on your sytem with brew, conda or apt-get depending on your system. Cf. remark from @papadop about having good defaults for BLA_VENDOR

**Comment by massich** [comment]
I understand. We should keep a list of things that are half way and take a
turn.
For this case, maybe we can do it outside of this PR with a commit message
with a keyword we can search for.

**Comment by massich** [comment]
Secon thought adding it to cmake/options.cmake with `XXX` should be enough.
You can add it first thing in options or in root cmakelisrs so that we'll
find it often and we won't forget. Otherwise you can put it in
compiler_options which is where it should be, but is more likely to pass
unnoticed

**Pull request merged**